provisioner/ansible/provisioner.go

```diff
@@ -111,6 +112,13 @@ func (p *Provisioner) Prepare(raws ...interface{}) error {
 		p.config.LocalPort = "0"
 	}
 
+	if p.config.User == "" {
+		p.config.User = os.ExpandEnv("${USER}")
```

Comment:
Just remembered we can ignore Windows in this case since [it is not supported to run the _Controller_ on a Windows host](http://docs.ansible.com/ansible/intro_windows.html#reminder-you-must-have-a-linux-control-machine).